kohese-portal/tsconfig.json

```diff
@@ -14,8 +14,6 @@
     "noImplicitAny": false,
     "types": [
       "core-js",
-      "socket.io",
-      "socket.io-client",
```

Comment:
socket.io and socket.io-client contains its own types since it has been rewritten in typescript. These type definitions are no longer necessary and in some cases may even causes errors.